gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-core/src/main/resources/webroot/views/login_callback_device_identifier.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Login</title>
+</head>
+
+<body>
+    <form id="callbackForm" th:action="${action}" method="post">
+    </form>
+    <script th:if="${deviceIdentifierProvider == 'FingerprintJsV3Pro'}" th:src="@{../assets/js/remember-device/fingerprintjs-v3-pro.js}"></script>
+    <script th:if="${deviceIdentifierProvider == 'FingerprintJsV3Pro'}" th:attr="nonce=${script_inline_nonce}">
+        (function() {
+            loadFingerprintJsV3Pro("[[${fingerprint_js_v3_pro_browser_token}]]", "[[${fingerprint_js_v3_pro_region}]]", fp => {
+                if (fp.visitorId) {
+                    let inputDeviceId = document.createElement("input");
+                    inputDeviceId.setAttribute("type", "hidden");
+                    inputDeviceId.setAttribute("name", "deviceId");
+                    inputDeviceId.value = fp.visitorId;
+                    document.getElementById('callbackForm').appendChild(inputDeviceId);
+                }
+                document.getElementById('callbackForm').submit();
+            });
+        })();
+    </script>
+
+    <script th:if="${deviceIdentifierProvider == 'FingerprintJsV3Community'}" th:src="@{../assets/js/remember-device/fingerprintjs-v3.js}"></script>
+    <script th:if="${deviceIdentifierProvider == 'FingerprintJsV3Community'}" th:attr="nonce=${script_inline_nonce}">
```

Comment:
When activating CSP with the correct directives
`nonce` is not set 

It is done properly though on the login page

Login Page
<img width="368" alt="image" src="https://user-images.githubusercontent.com/8531515/218969272-c1b0e1b4-2a5a-483c-9f49-65b4f6cc07eb.png">

Login callback page
<img width="582" alt="image" src="https://user-images.githubusercontent.com/8531515/218967818-58e7f2f6-f585-4dd8-b931-04eaca0b59de.png">